### PR TITLE
132 improved descriptiveness of local deck validation issues

### LINKED ISSUE #117: Add class docstrings

### LINKED ISSUE #132: improved descriptiveness of local deck validation issues

### PR DESCRIPTION
**Pull request by whatdoes3plus1equalsto**
closes #117 
closes #132

### CODE REVIEW

**Comment by whatdoes3plus1equalsto** [comment]
my bad, should've seperate the header changes, if you want to look at deck validator changes, please see IDeckValidator.ts and DeckValidator.ts

**Comment by whatdoes3plus1equalsto** [comment]
> my bad, should've seperate the header changes, if you want to look at deck validator changes, please see IDeckValidator.ts and DeckValidator.ts

or simply go to the first commit

File: src/objects/deck/DeckValidator.ts
Comment (VanJackal) [inline src/objects/deck/DeckValidator.ts:1]: where is DeckIssues defined?

Comment (whatdoes3plus1equalsto) [reply src/objects/deck/DeckValidator.ts:1]: IDeckValidator.ts

**Review by VanJackal — APPROVED** [review]
the fact js doesnt have a proper equivalent to printf is kinda wild

**Pull request merged**